src/auro-checkbox-group.scss

```diff
@@ -35,6 +31,16 @@ legend {
   margin-bottom: var(--auro-size-md);
 }
 
-.errorText {
-  color: var(--auro-color-text-error-on-light)
+.selectElement-helpText {
+  margin: var(--auro-size-50, .25rem) 0;
+  color: var(--auro-color-text-secondary-on-light);
+  font-size: var(--auro-text-body-size-xs);
+  line-height: 1rem;
+}
+
+:host([validity]:not([validity='valid'])) {
+  .selectElement-helpText {
+    color: var(--auro-color-alert-error-on-light);
+  }
```

Comment:
This update and the removal of the `:host([error]) {}` selector is a change from what is in the Figma components that the designers are using. This will need to be addressed before we can release this. 

<img width="908" alt="Screen Shot 2022-11-21 at 3 09 51 PM" src="https://user-images.githubusercontent.com/181089/203176531-14884720-cb08-476c-97ea-e7867fc12968.png">

Reply:
There is no change in behavior here because applying the `error` attribute results in the `validity` attribute being applied and the styles all work out as they should. There is no change in the expected API use of the error attribute from the customers perspective.

Reply:
Here is a screenshot from the demo page per the updates. The error(help text) is smaller and the label is NOT red. 

<img width="968" alt="Screen Shot 2022-11-21 at 4 21 54 PM" src="https://user-images.githubusercontent.com/181089/203184530-c2945f89-2d9f-4ea8-9893-1bac1a5597d8.png">